source/clear-linux/tutorials/multi-boot/multi-boot.rst

```diff
@@ -35,6 +38,23 @@ multi-booting computer is as follows:
    This process is not guaranteed to work with all Linux distributions and
    all their versions.
 
+Detailed procedures
+*******************
+
+.. toctree::
+   :maxdepth: 2
+
+   multi-boot-cl
+   multi-boot-win
+   multi-boot-rhel
+   multi-boot-sles
+   multi-boot-ubuntu
+   multi-boot-mint
+   multi-boot-reset-bl
+
```

Comment:
1. I'd prefer to put the "Detailed procedures" section after the "Test operating systems".  The "Tested operating systems" table shows the configurations of the example OSes for the tutorial.  

2. The "multi-boot-cl" SHOULD be show in full instead of a link you can jump to.  It's a prerequisite before other OSes can be installed.  Then the other OSes can be links.